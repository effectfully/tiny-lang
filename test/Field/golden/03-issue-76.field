ext x : field;
for z = 0 to -1 do
    let x : field = x;
end;
let y : field = if (0 == x) then 0 else 0;
