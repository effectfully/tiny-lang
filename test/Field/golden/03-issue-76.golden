ext x_0 : field;
let y_1 : field = if (0 == (x_0 : field)) then 0 else 0;
